src/app/clusters/push-av-stream-transport-server/push-av-stream-transport-server.cpp

```diff
@@ -0,0 +1,1133 @@
+        ChipLogError(Zcl, "HandleManuallyTriggerTransport[ep=%d]: Invalid Transport status",
+                     AttributeAccessInterface::GetEndpointId().Value());
+        ctx.mCommandHandler.AddClusterSpecificFailure(ctx.mRequestPath, clusterStatus);
+        return;
+    }
+    if (transportConfiguration->transportOptions.HasValue())
+    {
+        if (transportConfiguration->transportOptions.Value().triggerOptions.triggerType == TransportTriggerTypeEnum::kContinuous)
+        {
+
+            auto clusterStatus = to_underlying(StatusCodeEnum::kInvalidTriggerType);
+            ChipLogError(Zcl, "HandleManuallyTriggerTransport[ep=%d]: Invalid Trigger type",
+                         AttributeAccessInterface::GetEndpointId().Value());
+            ctx.mCommandHandler.AddClusterSpecificFailure(ctx.mRequestPath, clusterStatus);
+            return;
+        }
+        if (transportConfiguration->transportOptions.Value().triggerOptions.triggerType == TransportTriggerTypeEnum::kCommand &&
+            timeControl.HasValue() == false)
+        {
+
+            ChipLogError(Zcl, "HandleManuallyTriggerTransport[ep=%d]: Time control field not present",
+                         AttributeAccessInterface::GetEndpointId().Value());
+            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::DynamicConstraintError);
+            return;
+        }
+    }
+
+    // When trigger type is motion in the allocated transport but triggering it manually
+    if (timeControl.HasValue() == false)
+    {
+        timeControl = transportConfiguration->transportOptions.Value().triggerOptions.motionTimeControl;
+    }
+
+    // Call the delegate
+    status = mDelegate.ManuallyTriggerTransport(connectionID, activationReason, timeControl);
+
+    if (status == Status::Success)
+    {
+        mDelegate.GeneratePushTransportBeginEvent(connectionID, TransportTriggerTypeEnum::kCommand, MakeOptional(activationReason));
+    }
+
+    ctx.mCommandHandler.AddStatus(ctx.mRequestPath, status);
+}
+
+void PushAvStreamTransportServer::HandleFindTransport(HandlerContext & ctx,
+                                                      const Commands::FindTransport::DecodableType & commandData)
+{
+    Commands::FindTransportResponse::Type response;
+
+    Optional<DataModel::Nullable<uint16_t>> connectionID = commandData.connectionID;
+
+    std::vector<TransportConfigurationStruct> transportConfigurations;
+
+    if ((connectionID.HasValue() == false) || connectionID.Value().IsNull())
+    {
+        if (mCurrentConnections.size() == 0)
+        {
+            ChipLogError(Zcl, "HandleFindTransport[ep=%d]: ConnectionID not found",
+                         AttributeAccessInterface::GetEndpointId().Value());
+            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::NotFound);
+            return;
+        }
+
+        for (auto & connection : mCurrentConnections)
+        {
+            if (connection.fabricIndex == ctx.mCommandHandler.GetAccessingFabricIndex())
```

Comment:
https://github.com/project-chip/connectedhomeip/pull/37787#discussion_r2092220302 did not get addressed (or at least I don't see a link to the spec issue).

In general, links to these spec issues should be in the code, so we know we need to update it once those are fixed.